components/infobox/wikis/counterstrike/infobox_league_custom.lua

```diff
@@ -176,7 +177,7 @@ function CustomInjector:parse(id, widgets)
 			Cell{
 				name = Template.safeExpand(mw.getCurrentFrame(), 'Valve/infobox') .. ' Tier',
 				content = {self.caller:_createValveTierCell()},
-				classes = {'valvepremier-highlighted'}
+				classes = {HighlightConditions.tournament(args) and 'valvepremier-highlighted' or ''}
```

Comment:
wouldn't we need some standardization of the input data?
- key valvetier vs publishertier
- capitalization of the input value?
- ...

like i think we need parent.data instead of args passed here